test-project/Assets/EditmodeTests/GameObjectRepresentation/Readers/ReaderWriterTestsBase.cs

```
@@ -31,10 +32,10 @@ public void TearDown()
             world.Dispose();
         }
 
-        protected struct SomeOtherComponent : IComponentData, ISpatialComponentData
+        protected struct SomeOtherComponent : ISpatialComponentData, IComponentData
```

🤔

---

Yeah this was weird - I'm scared that this was silently failing for ages..?

---

I would not expect this to fail though unless we have some interesting linting things

---

Wait I thought this comment was about something else - this was me running the linter over all the tests 😉